### PR TITLE
WW-494 | restyle description textarea

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/WW-494

@Wikia/west-wing @Warkot

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 1]
```diff
@@ -161,23 +161,20 @@
...
 		transition: height 250ms;
 		width: $cnw-textarea-width;
-
-		&.active {
-			height: 8 * $wds-line-height;
-		}
 	}
 
 	.desc-block {
 		display: flex;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Do we still need `flex`?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 2]
```diff
@@ -161,24 +161,16 @@
 	}
 
 	textarea {
-		height: 2 * $wds-line-height;
+		height: 4 * $wds-line-height;
 		overflow-x: hidden;
 		overflow-y: auto;
 		resize: none;
 		transition: height 250ms;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Should we remove also `transition` and code from js which adds `active` class?

Comment (rybmat) [reply extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: I thought I removed it... :p

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/23e9509c65a36c84b72aebc20fb9fed5095ca060/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
2. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/23e9509c65a36c84b72aebc20fb9fed5095ca060/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
3. MAJOR [WikiBuilder.js#L475](https://github.com/Wikia/app/blob/23e9509c65a36c84b72aebc20fb9fed5095ca060/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L475): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
4. MAJOR [WikiBuilder.js#L609](https://github.com/Wikia/app/blob/23e9509c65a36c84b72aebc20fb9fed5095ca060/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L609): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
5. MAJOR [WikiBuilder.js#L611](https://github.com/Wikia/app/blob/23e9509c65a36c84b72aebc20fb9fed5095ca060/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L611): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Review by lukaszkonieczny — APPROVED** [review]

**Review by Warkot — APPROVED** [review]

**Pull request merged**